ledger/src/blockstore_processor.rs

```diff
@@ -1725,18 +1725,21 @@ fn process_next_slots(
     blockstore: &Blockstore,
     leader_schedule_cache: &LeaderScheduleCache,
     pending_slots: &mut Vec<(SlotMeta, Bank, Hash)>,
-    halt_at_slot: Option<Slot>,
+    opts: &ProcessOptions,
```

Comment:
don't love the name, but I know you're just maintaining the existing convention, so I'll hold my nose